src/MyParcelComApi.php

```diff
@@ -837,17 +839,38 @@ private function arrayToFilter(array $array)
     {
         $filters = [];
         foreach ($array as $name => $value) {
-            /**
-             * @deprecated This condition is to keep the SDK backwards
-             * compatible until the API has been updated to use the new filter.
-             */
-            if ($name === 'has_active_contract') {
-                $filters['filter[has_contract]'] = $value;
-            }
-
             $filters["filter[$name]"] = $value;
         }
 
         return $filters;
     }
+
+    /**
+     * @param bool                  $onlyActiveContracts
+     * @param null|CarrierInterface $specificCarrier
+     * @return array
+     */
+    private function determineCarriers($onlyActiveContracts, $specificCarrier = null)
```

Comment:
Missing typehints and return type

Reply:
The SDK supports PHP 5.6. So there is no support for return types or scalar type hints.